src/components/SideNav.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Jumbotron, Grid, Row, Col, Image, Button } from 'react-bootstrap';
+import './SideNav.css';
+
+export default class SideNav extends Component {
+  constructor(){
+    super();
+    this.state = {
+      isLoading: true,
+      category: []
+    }
+  }
+
+  componentWillMount() {
+    this.fetchData();
+  }
+
+  submitCategory(event) {
+    console.log(this)
+    fetch(`https://api.mercadolibre.com/categories/${this}`, {
+        type: 'GET',
+        datatype: 'json'
+      }).then(response => response.json())
+          .then(data => data.children_categories.map(sub=>(
+           {
+            subCategories: `{sub.name}`,
+            subId: `{sub.id}`,
+           }
+
+         )))
+        .then(subCategory => this.setState({
```

Comment:
Cuando hago click en una sub categoria en `https://mercury-50142.firebaseapp.com/#`, el siguente error sucede: `TypeError: t.setState is not a function`.